### PR TITLE
Added s04e03 shownotes

### PR DESCRIPTION
**Pull request by nixi-awoo**
This is my very first take on making shownotes - feedback is welcome.

Please check line 46 where there are several relevant links. 
Will that work or can I only post one link per line?

### CODE REVIEW

File: s04e03/episode.txt [hunk 1]
```diff
@@ -12,4 +12,74 @@ ADDITIONAL:
...
+	Guns of Icarus Online http://steamcommunity.com/groups/steamlug#events/92690048508271209
+00:05:50 @HeroOhOne got his Thrustmaster TX xbone wheel initialized in Linux https://github.com/her001/tmdrv
+
+*Steam/Valve News*
+00:07:16 Steam bug which did not allow Chromium-based games or applications to be launched from the Linux client has been fixed http://steamcommunity.com/groups/SteamClientBeta#announcements/detail/903337784334862903
+00:08:01 Lunar New Year Sale http://store.steampowered.com/news/20313/
+
+*Tech News*
+00:09:15 AMD GPUOpen launched http://gpuopen.com/welcometogpuopen/
+	At least 30 git repos spread over 4 github organizations https://github.com/GPUOpen-LibrariesAndSDKs, https://github.com/GPUOpen-Effects, https://github.com/GPUOpen-Tools, https://github.com/GPUOpen-ProfessionalCompute-Tools
```
Comment (nixi-awoo) [inline s04e03/episode.txt]: I was not able to find a different shownotes file with more than one link on the same line. 
Will this work?

**Comment by johndrinkwater** [comment]
Hey, this is awesome work! Been a bit of a tiring day, so will just comment on your comment and if no one else checks this out before me, will look tomorrow

File: s04e03/episode.txt (hunk 1 above)
Comment (johndrinkwater) [inline s04e03/episode.txt]: Remove the commas and it will be fine, we just list naked URLs separated by spaces for multiples.

**Comment by her001** [comment]
After skimming it, I noticed a few things:
- Cohost names need to be consistent (always "@DerRidda" "Dora (@hmccio)" "@HeroOhOne")
- Proper nouns should all be written/capitalized the official way (like "GitHub")
- Each title of a section should have a timestamp as well, including sign-off.
- You should probably squash all those commits into one and change the message to say e3 instead of e1

The way you have line 46 right now is good.

**Comment by nixi-awoo** [comment]
Silly me, I named the pull request e01 by mistake. I am going to make a fresh pull request with all those changes.

**Comment by Tele42** [comment]
Commits can be squashed, changed, completely replaced by pushing the adjustment into your pull request git branch, this pull request can be renamed with the edit button on the same line as the title.

**Comment by her001** [comment]
...Thinking about it now, the fact that my wheel works on Xbox One is completely irrelevant. It isn't an exclusively Xbox wheel, and so including that is almost misleading.

I'd remove the mention of it.

**Comment by Tele42** [comment]
It seems fine as presented since there is another, easily confused model and that is one way to tell them apart.

**Comment by her001** [comment]
There are two Thrustmaster TX wheels, but they have the same base (the motor and such) and will work exactly the same on Linux. There are many Ferrari 458 wheels by Thrustmaster, but as that is not even mentioned, I do not see where the confusion will arise.

On the extreme other side, it could look like I did some voodoo magic to make Xbox exclusive controllers work on Linux, which is obviously not the case.

**Comment by nixi-awoo** [comment]
Even though it could be considered an arbitrary nitpicking, I understand what @her001 means.
After all, I am (usually) a perfectionist as well.

**Comment by johndrinkwater** [comment]
Beyond a squashing of the commits, this seems mergable - would you do that and force push an update to your repo, it’ll sync on this PR.

**Pull request merged**